src/loaders/ccloudResourceLoader.ts

```diff
@@ -67,6 +67,10 @@ export class CCloudResourceLoader extends ResourceLoader {
     }
     super();
 
+    this.registerEventListeners();
+  }
+
+  private registerEventListeners(): void {
```

Comment:
Moved the ccloud loader event listener(s) into own method, back from when thinking would need to register a new event listener for websocket events. That didn't end up being the case. No harm either way.